platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -137,5 +137,8 @@ public record PlatformSchedulersConfig(
         @ConfigProperty(defaultValue = "SEQUENTIAL CAPACITY(60) UNHANDLED_TASK_METRIC")
                 TaskSchedulerConfiguration stateGarbageCollector,
         @ConfigProperty(defaultValue = "200ms") Duration stateGarbageCollectorHeartbeatPeriod,
+        @ConfigProperty(defaultValue = "SEQUENTIAL CAPACITY(60) UNHANDLED_TASK_METRIC")
+                TaskSchedulerConfiguration signedStateSentinel,
+        @ConfigProperty(defaultValue = "200ms") Duration signedStateSentinelHeartbeatPeriod,
```

Comment:
It's probably safe to make this a very infrequent sort of thing. It's not important we log this warning immediately, just eventually. Perhaps this thread can wake up once per minute?

Reply:
setting to 1000ms

Reply:
You can instead do `1s`